GoogleForCreators/web-stories-wp
# Editor: Added keyboard shortcut menu

## Summary

This PR adds the keyboard shortcuts button and popup menu to the editor.

## User-facing changes

The "Keyboard Shortcuts" toggle appears about the safe zone toggle in the editor:
<img src="https://user-images.githubusercontent.com/40646372/92637726-17651600-f28e-11ea-9593-17f1dbad55ae.png" height="100">

## Testing Instructions

1.) Open a new or existing story in the Editor.
2.) Using your mouse, hover over the "Keyboard Shortcuts" button, notice its informative tooltip.
3.) Click on the button, notice the pop up with helpful shortcuts.

Other ways to interact with the "Keyboard Shortcuts" menu:
-  Hit `Cmd` + `/` to Open/Close the menu
-  Hit `ESC` to close the menu if it's open.
- Click off the menu to close it if it's open.

All the shortcuts in the menu should work. So feel free to test them out.  

**NOTE:**  The shortcut keys to "Switch between element, workspace and design panels" will _not_ work until #4241 is merged in.  It's currently in review so I felt it was okay to enable this section of shortcuts.  Every other shortcut should work as described.

Fixes #359 

## Screenshots:

Shortcut modal on Mac (`⌘`):
<img src="https://user-images.githubusercontent.com/40646372/92640467-2c43a880-f292-11ea-860f-a76c1228e4f9.png" height="300">

Modal on PC (`Ctrl`):
<img src="https://user-images.githubusercontent.com/40646372/92640643-72990780-f292-11ea-8383-107a5f061113.png" height="300">
